pkgs/development/python-modules/casadi/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  stdenv,
+  buildPythonPackage,
+  lib,
+  fetchPypi,
+  patchelf
+}:
+
+buildPythonPackage rec {
+  pname = "casadi";
+  version = "3.6.5";
+  format = "wheel";
```

Comment:
casadi wheel is vendoring a ton of stuff. it would probably be better to build it from source.

Reply:
I'm working on it: https://github.com/nim65s/nixpkgs/tree/casadi